.github/ISSUE_TEMPLATE/4-chore.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Chore
+about: Work on a chore needing completion
+labels: 'type: chore'
+---
+
+<!--
+Please use this template to create your chore issue. You can use this template if you spot an out-of-date README, discover a misspelling, or happen upon a deeply nested 7-layer for-loop that could be better handled another way. Please use this template for your non-bug related fixes/updates/refactors.
+
+- Questions should be posted to: https://community.grafana.com
+- Use query inspector to troubleshoot issues: https://bit.ly/2XNF6YS
+- How to record and attach gif: https://bit.ly/2Mi8T6K
+-->
+
+**What is the chore?**:
+
+**Is there anything else we need to know?**:
+
+**Environment**:
```

Comment:
I think that we don't need the `Environment` part because most chores aren't tied to a Grafana version or datasource type etc, so these might end up being empty most of the times. What do you say?

Reply:
True; I think the environment is most helpful with debugging; I'll remove it.